<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="serialReceiveSendGroupBox">
        <property name="title">
         <string>串口收发</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="serialReceiveSendGroupBoxLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="serialSendLabel">
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>发送消息:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="serialRecieveLabel">
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>接收消息:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="isNewLineCheckBox">
             <property name="text">
              <string>发送新行</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="isAddTimeCheckBox">
             <property name="text">
              <string>时间戳</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="serialClearSendButton">
             <property name="text">
              <string>清除发送</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="serialClearReceiveButton">
             <property name="text">
              <string>清除接受</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QTextEdit" name="serialReceiveTextEdit">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLineEdit" name="serialSendLineEdit"/>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="serialClearReceiveSendButton">
             <property name="text">
              <string>清除收发</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="serialSingleSendButton">
             <property name="text">
              <string>单次发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="serialSettingsAction"/>
   <addaction name="systemSettingsAction"/>
  </widget>
  <widget class="QDockWidget" name="serialSettingsDockWidget">
   <property name="windowTitle">
    <string>串口设置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="serialSettingsWidget">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="serialSettingsLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="serialPortLabel">
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="serialPortComboBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="serialDisconnectButton">
         <property name="text">
          <string>断开</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/image/serialDisconnect</normaloff>:/image/serialDisconnect</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="serialBandRateComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="serialConnectButton">
         <property name="text">
          <string>连接</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/image/serialConnect</normaloff>:/image/serialConnect</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="serialBandRateLabel">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="serialRefreshButton">
         <property name="text">
          <string>刷新</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/image/serialRefresh</normaloff>:/image/serialRefresh</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="NotingLabel">
         <property name="text">
          <string>广告位招商</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="systemSettingsDockWidget">
   <property name="windowTitle">
    <string>系统设置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="systemSettingsWidget">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="systemSettingsLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="referencePathPointNumLabel">
         <property name="text">
          <string>参考路径点个数:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="currentSendReferencePathPointNumLabel">
         <property name="text">
          <string>当前发送</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="autorunPathPointNumLabel">
         <property name="text">
          <string>自动路径点个数:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="referencePathPointValueLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="autorunPathPointValueLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="currentSendReferencePathPointValueLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="currentSendAurorunPathPointValueLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="currentSendAurorunPathPointNumLabel">
         <property name="text">
          <string>当前发送</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="sendTimeIntervalLabel">
         <property name="text">
          <string>发送间隔(ms)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="chooseReferencePathFileButton">
         <property name="text">
          <string>选择参考路径</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QComboBox" name="intervalSendTimeComboBox"/>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QPushButton" name="chooseAutorunPathFileButton">
         <property name="text">
          <string>选择自动路径</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QPushButton" name="continuousSendAutorunPathPointButton">
         <property name="text">
          <string>连续发送自动路径</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QPushButton" name="singleSendAutorunPathPointButton">
         <property name="text">
          <string>单次发送自动路径</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QPushButton" name="resetSendAutorunPathButton">
         <property name="text">
          <string>复位发送自动路径</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="continuousSendReferencePathPointButton">
         <property name="text">
          <string>连续发送参考路径</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="singleSendReferencePathPointButton">
         <property name="text">
          <string>单次发送参考路径</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="resetSendReferencePathButton">
         <property name="text">
          <string>复位发送参考路径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="serialSettingsAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/serialSettings</normaloff>:/image/serialSettings</iconset>
   </property>
   <property name="text">
    <string>serialSettings</string>
   </property>
  </action>
  <action name="systemSettingsAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/systemSettings</normaloff>:/image/systemSettings</iconset>
   </property>
   <property name="text">
    <string>systemSettings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
